clientv3/integration/cluster_test.go

```diff
@@ -178,6 +178,27 @@ func TestMemberAddForLearner(t *testing.T) {
 	if resp.Member.IsLearner != isLearner {
 		t.Errorf("Added a member with IsLearner = %v, got %v", isLearner, resp.Member.IsLearner)
```

Comment:
I did not notice that a code block was moved from `TestMemberAddForLearner` to `TestMemberPromoteForLearner` in this PR.

Reply:
I'll send a PR to fix this, and also mitigating the race in test tomorrow.